src/OrderProvider.cs

```diff
@@ -693,6 +769,30 @@ private void ProcessOrderRow(Mapping mapping, ColumnMappingCollection columnMapp
                 row[SourceColumnNameForDestinationOrderCustomerAccessUserId] = accessUserId;
             }
         }
+        if (!string.IsNullOrEmpty(SourceColumnNameForDestinationOrderDeliveryAddressId))
+        {
+            object addressId = DBNull.Value;
+            var OrderDeliveryAddressExternalIdMapping = columnMappings.Find(cm => string.Compare(cm.DestinationColumn.Name, OrderDeliveryAddressExternalId, true) == 0);
```

Comment:
consider lazy loading or make a class variable to store the needed value since this code for Find will be called per each row processing so it will be time consuming

Reply:
ok, moved them out, so they are only called once for every table mapping